src/io/flutter/inspector/EvalOnDartLibrary.java

```diff
@@ -132,33 +132,31 @@ public void dispose() {
   public CompletableFuture<InstanceRef> eval(String expression, Map<String, String> scope, InspectorService.ObjectGroup isAlive) {
     return addRequest(isAlive, () -> {
       final CompletableFuture<InstanceRef> future = new CompletableFuture<>();
-      libraryRef.thenAcceptAsync((LibraryRef ref) -> {
-        vmService.evaluate(
-          getIsolateId(), ref.getId(), expression,
-          scope, true,
-          new EvaluateConsumer() {
-            @Override
-            public void onError(RPCError error) {
-              future.completeExceptionally(new RuntimeException(error.getMessage()));
-            }
-
-            @Override
-            public void received(ErrorRef response) {
-              future.completeExceptionally(new RuntimeException(response.toString()));
-            }
-
-            @Override
-            public void received(InstanceRef response) {
-              future.complete(response);
-            }
-
-            @Override
-            public void received(Sentinel response) {
-              future.completeExceptionally(new RuntimeException(response.toString()));
-            }
+      libraryRef.thenAcceptAsync((LibraryRef ref) -> vmService.evaluate(
```

Comment:
this change block is whitespace only, right?

Reply:
Yeah, it's just the built-in IntelliJ refactoring to prefer expression lambdas.